web/yo/app/scripts/directives/reviewpanel.js

```diff
@@ -399,7 +399,13 @@ angular.module('oncokbApp')
                             break;
                         case 'PROGNOSTIC_IMPLICATION':
                             uuid = $scope.tumor.progImp_uuid;
-                            items = [{obj: $scope.tumor.progImp, reviewObj: $scope.tumor.progImp_review, uuid: $scope.tumor.progImp_uuid}];
+                            items = [{obj: $scope.tumor.progImp, reviewObj: $scope.tumor.progImp_review, uuid: $scope.tumor.progImp_uuid},
+                                {obj: $scope.tumor.progLevel, reviewObj: $scope.tumor.progLevel_review, uuid: $scope.tumor.progLevel_uuid}];
```

Comment:
Oh, if we name this progLevel, then we should use diagLevel or progImpLevel and diagImpLevel. Either way.